linkerd/app/inbound/src/lib.rs

```diff
@@ -42,11 +42,12 @@ pub struct Config {
     pub profile_idle_timeout: Duration,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub struct Inbound<S> {
     config: Config,
     runtime: ProxyRuntime,
     stack: svc::Stack<S>,
+    tls_detect_metrics: metrics::tls_detect::ErrorRegistry,
```

Comment:
i thought about doing this initially but didn't want to make the diff huge from touching every place we destructure this struct...i guess since it's also needed in the direct stack, this was necessary after all.

Reply:
Yeah, actually it might be best to revert this as it breaks some other tests -- unless this impls `Default`?